.github/workflows/publish-qa.yml

```diff
@@ -48,10 +48,10 @@ jobs:
           file: ./Dockerfile
           push: true
           tags: |
-            nypl/patron-web:beta
-            nypl/patron-web:beta.${{ steps.get_sha.outputs.short_sha }}
-            nypl/patron-web:qa
-            nypl/patron-web:qa.${{ steps.get_sha.outputs.short_sha }}
+            thepalaceproject/web-patron:beta
```

Comment:
It might be useful to eventually (i.e., not necessarily on this PR) align the approach/naming conventions here and those used by [tpp/library-registry](https://github.com/ThePalaceProject/library-registry/blob/10675b7677677f289c958b3e2a7fe6efd585d9f7/.github/workflows/test-build.yml#L47-L91) and [tpp/circulation](https://github.com/ThePalaceProject/circulation/blob/f79fd8675721c164f0835d07eb21623d6be27719/.github/workflows/test-build.yml#L243-L318).

Note especially that:
- we are [using the GitHub Container Registry (GHCR)](https://github.com/ThePalaceProject/circulation/blob/f79fd8675721c164f0835d07eb21623d6be27719/.github/workflows/test-build.yml#L266-L271), rather than Docker Hub
- the [order/priority of image tags](https://github.com/ThePalaceProject/circulation/blob/f79fd8675721c164f0835d07eb21623d6be27719/.github/workflows/test-build.yml#L303-L307).

Reply:
Thanks, I wasn't sure what we were doing with docker images. Will address this when I enable CI.